### DISCUSSION TITLE
Raspberry Pi 4 with 2 HDMI output

### DISCUSSION THREAD

**Discussion by reinhardkrakow** [raysan5/raylib#2469] [Q&A]
Hi,
I am wondering if it is possible to write a program on Rasperry Pi 4 using raylib and outputs 2 separate graphics drawing on the 2 screens.

BR
Reinhard

**Comment by raysan5** [comment]
@reinhardkrakow Probably there is some way to do it but not out-of-the-box, it will require some investigation and probably some internal raylib modification. As a first step, you can render to two different RenderTextures, one of the could be directly rendered to default output while the other one could be converted into an Image and send to a custom framebuffer. It's just an idea, no tested.

**Reply by reinhardkrakow** [reply]
OK, I will try something. I let you know.
